Makefile

```diff
@@ -149,7 +149,8 @@ admission: init
 adm_image: admission
 	@echo "building admission controller docker images"
 	@cp ${ADMISSION_CONTROLLER_BIN_DIR}/${POD_ADMISSION_CONTROLLER_BINARY} ./deployments/image/admission
-	docker build ./deployments/image/admission -t ${REGISTRY}/yunikorn:admission-${VERSION}
+	docker build ./deployments/image/admission -t ${REGISTRY}/yunikorn:admission-${VERSION} \
```

Comment:
Just noticed that admission and scheduler are sharing the same repo and type(scheduler/admission) is included in the version, is this a common solution to make sure they are coming from the same version? 
I noticed repository is not consistent with the updated scheduler repo(${REGISTRY}/yunikorn:scheduler-${VERSION} ), we should update helm-charts/values.yaml as the scheduler repo changed, right?

Reply:
yes, we need to have them using same repo/version. we have some other PRs pending to merge for the repo updates. basically, we are migrating all docker images to ASF repo.